aggregator/options.go

```diff
@@ -146,7 +153,7 @@ func NewOptions() Options {
 		entryCheckInterval:        defaultEntryCheckInterval,
 		entryCheckBatchPercent:    defaultEntryCheckBatchPercent,
 		maxTimerBatchSizePerWrite: defaultMaxTimerBatchSizePerWrite,
-		defaultPolicies:           defaultDefaultPolicies,
+		defaultPolicies:           defaultPolicies,
```

Comment:
hm `defaultDefaultPolicies` means the default value for the default policies, should we change it back?